### PR TITLE
add mesh gateway

### PR DESCRIPTION
**Pull request by kalantar**

### CODE REVIEW

File: docs/tutorials/autox/autox.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ kubectl label deployment httpbin app.kubernetes.io/version=1.0.0
 Next, we will configure and install the AutoX controller.
 
 ```bash
-helm install autox autox --repo https://iter8-tools.github.io/hub/ --version 0.1.6 \
+helm install autox autox --repo https://iter8-tools.github.io/iter8 --version 0.1.6 \
```
Comment (kalantar) [inline docs/tutorials/autox/autox.md:55]: @Alan-Cha @sriumcp We've restricted the version in all the autox examples but not others. Why?

**Comment by kalantar** [comment]
Why do we have a hard-coded reference to a commit id: https://github.com/iter8-tools/docs/blob/main/docs/tutorials/autox/autox.md?plain=1#L183

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -72,10 +72,9 @@ repository, you can amend your commit with the sign-off by running:
 
 The Iter8 project consists of the following repos.
 
-1. [iter8-tools/iter8](https://github.com/iter8-tools/iter8): source for the Iter8 CLI
-2. [iter8-tools/hub](https://github.com/iter8-tools/hub): source for Iter8 experiment and supplementary charts
-3. [iter8-tools/docs](https://github.com/iter8-tools/docs): source for Iter8 docs
-4. [iter8-tools/homebrew-iter8](https://github.com/iter8-tools/homebrew-iter8): Homebrew formula for the Iter8 CLI
+1. [iter8-tools/iter8](https://github.com/iter8-tools/iter8): source for the Iter8
```
Comment (sriumcp) [inline CONTRIBUTING.md:75]: Perhaps `Iter8 source code` and `Iter8 documentation` for the two repos?

**Comment by kalantar** [comment]
> Why do we have a hard-coded reference to a commit id: https://github.com/iter8-tools/docs/blob/main/docs/tutorials/autox/autox.md?plain=1#L183

removed

File: docs/tutorials/autox/autox.md (hunk 1 above)
Comment (sriumcp) [reply docs/tutorials/autox/autox.md:55]: It is generally a good idea to do version pinning like this (for e.g., if we release a new autox chart with breaking changes, this documentation will still be valid).

I am not sure why we are inconsistent.

**Review by sriumcp — CHANGES_REQUESTED** [review]
Minor changes requested.

File: CONTRIBUTING.md (hunk 2 above)
Comment (kalantar) [reply CONTRIBUTING.md:75]: done

**Review by sriumcp — APPROVED** [review]
lgtm

**Pull request merged**